lib/blocks.js

```diff
@@ -62,11 +108,12 @@ BlockController.prototype.transformBlock = function(block, info) {
     nonce: blockObj.header.nonce,
     bits: blockObj.header.bits.toString(16),
     difficulty: block.header.getDifficulty(),
-    chainwork: info.chainWork,
-    previousblockhash: blockObj.header.prevHash,
-    nextblockhash: this.node.services.bitcoind.getNextBlockHash(block.hash),
+    chainwork: info.chainwork,
```

Comment:
The all-lowercase values were originally from bitcoind rpc values. We may want to normalize to camelCase for bitcore-node, such as: https://github.com/braydonf/bitcore-node/blob/bitcoind/lib/services/bitcoind.js#L1501